bot/pagination.py

```diff
@@ -229,20 +229,36 @@ def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
                 or isinstance(user_, Member) and any(role.id in MODERATION_ROLES for role in user_.roles)
             )
 
-            return (
+            right_reaction = (
+                # Reaction is on this message
+                reaction_.message.id == message.id
+                # Reaction is one of the pagination emotes
+                and str(reaction_.emoji) in PAGINATION_EMOJI
+            )
+
+            whitelisted = (
                 # Conditions for a successful pagination:
                 all((
-                    # Reaction is on this message
-                    reaction_.message.id == message.id,
-                    # Reaction is one of the pagination emotes
-                    str(reaction_.emoji) in PAGINATION_EMOJI,
                     # Reaction was not made by the Bot
                     user_.id != ctx.bot.user.id,
+                    # It was the right reaction and Message
+                    right_reaction,
                     # There were no restrictions
                     no_restrictions
                 ))
             )
 
+            if whitelisted:
+                return True
+
+            elif right_reaction and user_.id != ctx.bot.user.id:
+                ctx.bot.loop.create_task(
+                    reaction_.message.remove_reaction(
+                        reaction_.emoji, user_
+                    )
+                )
+                log.debug(f"Got reaction: {reaction_.emoji} from non-whitelisted user")
```

Comment:
```suggestion
                log.debug(f"Got reaction: {reaction_.emoji} from non-whitelisted user, reaction deleted")
```
Some additional clarification would be nice.